users/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from .views import UserProfileAPIView, LoginAPIView
+
+app_name = "users"
+
+urlpatterns = [
+    path('', UserProfileAPIView.as_view(), name='user_profile'),
```

Comment:
It will be good if you use or ', or " in code.

Reply:
okay, done